kubectl-plugin/kubectl-waiops

```diff
@@ -1764,19 +1777,19 @@ then
     elif [ "${VERSION_AIOPSORCHESTRATOR}" == "3.3" ]; 
     then
         component-versions-33
-    elif [ "${VERSION_AIOPSORCHESTRATOR}" == "0.1" ]
+    elif [ "${VERSION_AIOPSORCHESTRATOR}" == "0.1" ];
     then
-        # for developer builds (v0.1), we will check against release-4.4 versions
-        component-versions-44
+        # for developer builds (v0.1), we will check against release-4.5 versions
+        component-versions-45
         echo ""
         echo "${red}${bold}NOTE: ${normal}Your Cloud Pak for AIOps install appears to be an internal dev version (v${VERSION_AIOPSORCHESTRATOR})."
-        echo "      The status-upgrade command checks the status of upgrades to v3.3, v3.4, v3.5, v3.6, v3.7, v4.1, v4.2, v4.3, and v4.4 only." 
+        echo "      The status-upgrade command checks the status of upgrades to v3.3, v3.4, v3.5, v3.6, v3.7, v4.1, v4.2, v4.3, v4.4, and v4.5 only." 
         echo "      Therefore, the upgrade checks below will be against v4.4 component versions."
         echo "      This may influence your results below if your dev build is not based on release-4.4.${normal}"
```

Comment:
We'll need to update this text to `release-4.5` instead of `release-4.4`

Reply:
Solved in [615fa88](https://github.com/IBM/cp4waiops-samples/pull/228/commits/615fa889d34665b4e4b12cc69abb07fb122aa806)